opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/consent/Xs2aConsentInitiate.java

```diff
@@ -0,0 +1,46 @@
+package de.adorsys.opba.protocol.xs2a.service.xs2a.consent;
+
+import de.adorsys.opba.protocol.xs2a.service.ContextUtil;
+import de.adorsys.opba.protocol.xs2a.service.dto.ValidatedHeadersBody;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.context.Xs2aContext;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.context.ais.Xs2aAisContext;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.dto.consent.ConsentInitiateHeaders;
+import de.adorsys.xs2a.adapter.service.AccountInformationService;
+import de.adorsys.xs2a.adapter.service.Response;
+import de.adorsys.xs2a.adapter.service.model.ConsentCreationResponse;
+import de.adorsys.xs2a.adapter.service.model.Consents;
+import org.flowable.engine.delegate.DelegateExecution;
+import org.slf4j.Logger;
+
+import static de.adorsys.opba.protocol.xs2a.constant.GlobalConst.CONTEXT;
+
+public interface Xs2aConsentInitiate {
```

Comment:
Suggest not to generify things until it is needed - there is no use for this interface now

Reply:
done